hw/ip/aes/lint/aes.waiver

```diff
@@ -4,7 +4,7 @@
 #
 # waiver file for aes
 waive -rules COMBO_READ_WRITE -location {aes_key_expand.sv} -regexp {'regular\[[0-9]+:?[0-9]*\]\[2:1|4:2|6:5\]' is read from within the same combinational process block} \
-      -comment "regular[*] is assigned in a for loop, regular[*][1|2|5] depends on regular[*][0|1|4]"
+      -comment "regular\[*\] is assigned in a for loop, regular\[*\]\[1|2|5\] depends on regular\[*\]\[0|1|4\]"
```

Comment:
Ascentlint does not like this, even in comments (it tries to interpret [1|2|5] as a command for some reason)..

Reply:
That's quite unexpected, thanks for fixing it.

Reply:
The waivers are normal TCL files. Strings with double quotes are expanded, while strings with curly braces are not. So if you want to avoid escaping just use curly braces for strings.

(See also http://www.beedub.com/book/2nd/tclintro.doc.html#2668)